zebedee-cms/src/test/java/com/github/onsdigital/zebedee/util/slack/StartUpAlerterImplTest.java

```diff
@@ -149,13 +147,10 @@ public void testNotifyLocked_noChannelsConfigured_shouldDoNothing() throws Excep
     }
 
     @Test
-    public void testNotifylocked_originalMessageNull_shouldThrowEx() throws Exception {
-        alerter = new StartUpAlerterImpl(slackClient, null, true, true);
-
-        RuntimeException ex = assertThrows(RuntimeException.class, () -> alerter.queueUnlocked());
+    public void testNotifylocked_originalMessageNull_shouldDoNothing() throws Exception {
```

Comment:
Rename test method maybe? We are testing notifying of unlocked
```suggestion
    public void testNotifyUnlocked_originalMessageNull_shouldDoNothing() throws Exception {
```

Reply:
In fact, is this test method not identical to the one at line 178?
`testNotifyUnlocked_originalMessagesNull_shouldNotSendUpdate`

Reply:
Possibly. Ill revisit the tests they're probably out of sync with the current version.